sacrebleu/color.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+
+
+_ENABLE = True
+if not sys.stdout.isatty() or os.name == 'nt':
```

Comment:
I actually tried it first but couldn't understand fully its benefits and decided to use this simple class instead of adding one more dependency just for colors. But if that works on Windows as well, we can switch to it if everybody is OK.

Reply:
I now tried on a virtualbox Win10 VM and unfortunately colorama does not seem to work there out of the box. They say that you also require the termcolor package.

Reply:
The `termcolor` package is not needed, it just gives you syntactic sugar and more color codes.
I don't have access to a Windows box, but it seems
`python -c 'import colorama;print("\033[31m some red text")'`
[just works](https://travis-ci.org/github/udapi/udapi-python/jobs/765305822#L124).
BTW: Travis-CI newly supports Windows, so we could try it after handling [test.sh](https://github.com/mjpost/sacrebleu/blob/master/test.sh) (we could omit it from the tests or try if it works with git-bash or rewrite it to Python).

Reply:
Ok got it. `colorama` should be imported before `sys`. And also, in `sacrebleu.py` we need to remove re-opening `sys.stdin` and `sys.stdout` (potentially only the latter), otherwise it does not work. I don't know if it's still necessary to re-open standard systems that way, I've never used that in my projects. It's probably dating back from Python 2.x?

Reply:
Figured out how to run things smoothly on Windows and OS X with travis, I'll try to push those as well to this branch.